mypy/fastparse2.py

```diff
@@ -637,6 +638,9 @@ def visit_ImportFrom(self, n: ast27.ImportFrom) -> ImportBase:
                            n.level,
                            [(a.name, a.asname) for a in n.names])
         self.imports.append(i)
+        if (n.module == '__future__' and len(n.names) == 1 and
```

Comment:
Will this work if I do `from __future__ import absolute_import, unicode_literals`? That works at runtime.

Also, I don't think the `asname` check is necessary. This program:
```
from __future__ import unicode_literals as why_would_you_do_this
print(type(''))
```
prints "unicode".